recipes/mgis/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "mgis" %}
+{% set version = "2.1" %}
+{% set mfront_ver = "4.1.0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  - fn: '{{ name }}_{{ version }}.tar.gz'
+    url: https://github.com/thelfer/MFrontGenericInterfaceSupport/archive/refs/tags/MFrontGenericInterfaceSupport-{{ version }}.tar.gz
+    sha256: f5b556aab130da0c423f395fe4c35d6bf509dd8fc958242f2e37ea788464aea9
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  skip: true  # [py2k or osx or win or (python_impl == 'pypy')]
+
+requirements:
+  build:
+    - "{{ compiler('c') }}"
+    - "{{ compiler('cxx') }}"
+    - "{{ compiler('fortran') }}"
+    - cmake >=3.26.4
+    - autoconf
+    - python
+    - make
+
+  host:
+    - python
+    - boost 1.78
+    - numpy 1.23
```

Rather use the global pinning.
```suggestion
    - numpy
```

---

Sure, I can let numpy use the global pinning for now, but a downstream dependency [code_aster](https://github.com/conda-forge/code-aster-feedstock) is picky wrt. numpy versions (it uses the c-api of numpy). So I will probably have to add support for multiple numpy versions once we've updated all the code_aster dependencies and start testing that